dispatches/models/nuclear_case/unit_models/test_hydrogen_turbine.py

```diff
@@ -0,0 +1,120 @@
+#############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Turbo-Generator Set for a Hydrogen turbine.
+
+Compressor -> Stoichiometric Reactor -> Turbine
+Author: Konor Frick
+Date: April 2, 2021
+Notes: it is noted that in this example the hydrogen is compressed along with the air in the compressor
+As opposed to having a separate fuel injection system. Noting this is a simplified version of the H2 turbine.
+"""
+import pytest
+from pyomo.environ import ConcreteModel, SolverFactory, \
+    value
+from idaes.core import FlowsheetBlock
+
+
+from dispatches.models.nuclear_case.properties.hturbine_ideal_vap \
+    import configuration
+import dispatches.models.nuclear_case.properties.h2_reaction \
+    as reaction_props
+from dispatches.models.nuclear_case.unit_models.\
+    hydrogen_turbine_unit import HydrogenTurbine
+from idaes.generic_models.properties.core.generic.generic_property \
+    import GenericParameterBlock
+
+
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+
+def test_build():
+    m = ConcreteModel()
+
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+
+    # Air Properties
+    m.fs.properties1 = GenericParameterBlock(default=configuration)
+
+    m.fs.reaction_params = reaction_props.\
+        H2ReactionParameterBlock(default={"property_package": m.fs.properties1})
+
+    # Adding H2 turbine model
+    m.fs.h2_turbine = HydrogenTurbine(
+                default={"property_package": m.fs.properties1,
+                         "reaction_package": m.fs.reaction_params})
+
```

Comment:
Should we adds some checks here that the turbine contains the expected components (compressor, reactor, turbine, arcs, conversion)?